students/grzegorz_bajorski/lesson_04_unit_testing/the_length_of_the_segment_tests.py

```diff
@@ -0,0 +1,35 @@
+import pytest
```

Comment:
module name should be in one of the following formats: test_*.py or *_test.py ("s" at the end is not needed)